tokio/src/io/registration.rs

```diff
@@ -34,9 +34,9 @@ cfg_io_driver! {
     /// stream. The write readiness event stream is only for `Ready::writable()`
     /// events.
     ///
-    /// [`new`]: #method.new
-    /// [`poll_read_ready`]: #method.poll_read_ready`]
-    /// [`poll_write_ready`]: #method.poll_write_ready`]
+    /// [`new`]: method@Self::new
+    /// [`poll_read_ready`]: method@Self::poll_read_ready`]
+    /// [`poll_write_ready`]: method@Self::poll_write_ready`]
```

Comment:
This seems like it has an extra \`] at the end.

Reply:
Good catch, i'll fix it, thanks!

Reply:
Fixed, thanks!